django_ibmi/schema.py

```diff
@@ -46,6 +41,18 @@ class DB2SchemaEditor(BaseDatabaseSchemaEditor):
     sql_drop_pk = "ALTER TABLE %(table)s DROP PRIMARY KEY"
     sql_drop_default = "ALTER TABLE %(table)s ALTER COLUMN %(column)s DROP DEFAULT"
 
+    def quote_value(self, value):
+        if isinstance(value, (datetime.date, datetime.time, datetime.datetime)):
+            return "'%s'" % value
+        elif isinstance(value, str):
+            return "'%s'" % value.replace("\'", "\'\'").replace('%', '%%')
+        elif isinstance(value, (bytes, bytearray, memoryview)):
+            return "'%s'" % value.hex()
```

Comment:
```suggestion
            return "x'%s'" % value.hex()
```

For hex literals, we'd need to put an 'x' before the quote